recipes/sfml/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
```

Comment:
Modern cmake, dont use ${PROJECT_NAME}, write trst_package instead

Reply:
?

Reply:
Here you go: https://www.reddit.com/r/cmake/comments/j4xysx/modern_cmake_tips/g7vl9nj/

Reply:
Honestly, in test_package it doesn't matter, and it has nothing to do with "Modern CMake".

Reply:
Being a good example is the cause here, but it's your call 🤙

Reply:
It's fine to use `PROJECT_NAME` with only one target, not sure that a comment on reddit is relevant ;)

Reply:
> Modern cmake, dont use ${PROJECT_NAME}, write test_package instead

That's the first time I've ever heard that one 😱 there's a few of top notch OSS, like curl, that do this so it's hard to say its a best practice.

Reply:
yea, and curl != modern ;-)